### PR TITLE
Support PHP 8 and End of Support 5.6, 7.0

### PR DESCRIPTION
**Pull request by moririnson**

### CODE REVIEW

**Comment by moririnson** [comment]
This changes will be released when next major version up.

**Review by satosby — APPROVED** [review]
LGTM

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     {
       "name": "Satoru Yoshihara",
       "email": "vaduz0@gmail.com",
-      "role": "Maintainer"
+      "role": "Retired"
```
Comment (Vaduz) [inline composer.json:21]: 😢

Comment (moririnson) [reply composer.json:21]: 😭

**Comment by moririnson** [comment]
thanks~

**Pull request merged**